data/mods/Magiclysm/monsters/zombified_monsters.json

```diff
@@ -32,5 +32,38 @@
     "death_function": [ "NORMAL" ],
     "upgrades": { "half_life": 28, "into_group": "GROUP_ZOLF_UPGRADE" },
     "flags": [ "SEES", "HEARS", "SMELLS", "STUMBLES", "WARM", "KEENNOSE", "BASHES", "POISON", "NO_BREATHE", "REVIVES", "FILTHY" ]
+  },
+  {
+    "id": "mon_zorc",
```

Comment:
I didn't see any of the orc mobs above with a `"zombify_into": "mon_zorc"`   line.